### PR TITLE
Enabling more video controls: Contrast, Hue, Sharpness, etc

### PR DESCRIPTION
**Pull request by tuyenld**
- Follow peterbay's docs: https://github.com/peterbay/uvc-gadget/blob/master/doc/src/video-controls.md
- Buildroot adding project-specific patches: https://buildroot.org/downloads/manual/manual.html#customize-patches

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -51,11 +51,11 @@ jobs:
...
-      - name: Cache Build
-        uses: actions/cache@v2
-        with:
-          path: output
-          key: ${{ matrix.raspberry }}-${{ env.BUILDROOT_DIR }}-${{ env.BUILDROOT_VERSION }}-${{ env.CACHE_VERSION }}
+#       - name: Cache Build
+#         uses: actions/cache@v2
+#         with:
+#           path: output
+#           key: ${{ matrix.raspberry }}-${{ env.BUILDROOT_DIR }}-${{ env.BUILDROOT_VERSION }}-${{ env.CACHE_VERSION }}
```
Comment (dgsiegel) [inline .github/workflows/release.yml:58]: This is not needed. We can clear the cache by updating `env.CACHE_VERSION` through the Github UI. I've done that today

**Comment by tuyenld** [comment]
Some settings were not exposed on Linux machine.

[webcamoid](https://webcamoid.github.io/) tested on Debian 10.

linux

[webcamoid](https://webcamoid.github.io/) tested on Window 7.

window-enable

**Comment by htruong** [comment]
So in order for this to merge:

- We need on `master`

**Comment by tuyenld** [comment]
@htruong 
This commit is already on master branch.
Do you think it is good to merge? Or wait until I find out why some controls are missing on Linux?

**Comment by htruong** [comment]
Tested and validated. LGTM. Merging! Woohoo!

**Pull request merged**